tests/test_symbolics.py

```diff
@@ -214,6 +214,12 @@ def test_extended_sympy_arithmetic():
     assert ccode(-1 + bar) == '-1 + o->bar'
 
 
+def test_integer_abs():
```

Comment:
If the conf issue mentioned here : https://github.com/devitocodes/devito/pull/2018/files#r994249271
is fixed, is it worth adding any GPU tests?